src/components/header.js

```diff
@@ -1,7 +1,129 @@
-import React from "react";
+import React, { useState } from "react";
+import LanguageSwitcher from "./LanguageSwitcher";
+import {
+  ExtendedNav,
+  GovBanner,
+  Header as HeaderUSWDS,
+  Link,
+  Menu,
+  NavDropDownButton,
+  NavMenuButton,
+  Search,
+  Title,
+} from "@trussworks/react-uswds"; // TODO: read through documentation of each component to see if used correctly
+import { withTranslation } from "react-i18next";
 
-const Header = () => {
-  return <div>Header</div>;
+const Header = ({ t, i18n }) => {
+  const govBannerLang = {
+    /* TODO: might want to keep in a separate file */
+    en: "english",
+    es: "spanish",
+  };
+
+  const [expanded, setExpanded] = useState(false);
+  const onClick = () => setExpanded((prvExpanded) => !prvExpanded);
+  const [isOpen1, setIsOpen1] = useState([false]);
+  const [isOpen2, setIsOpen2] = useState([false]);
+
+  const testMenuItemsOne = [
+    <Link href="#linkOne" key="one">
+     {t('nav.dropdownOne.simpleLinkOne')}
+    </Link>,
+    <Link href="#linkTwo" key="two">
+      {t('nav.dropdownOne.simpleLinkTwo')}
+    </Link>
+  ]; // TODO: href or to?
+
+  const testMenuItemsTwo = [
+    <Link href="#linkThree" key="one">
+      {t('nav.dropdownTwo.simpleLinkThree')}
+    </Link>,
+    <Link href="#linkFour" key="two">
+      {t('nav.dropdownTwo.simpleLinkFour')}
+    </Link>
+  ];
+  // TODO: href or to?
+
+  const testMenuItemsThree = [
+    <Link href="#linkFive" key="one">
+      {t('secondaryLinks.linkOne')}
+    </Link>,
+    <Link href="#linkSix" key="two">
+      {t('secondaryLinks.linkTwo')}
+    </Link>
+  ];
+  // TODO: href or to?
+
+  const testItemsMenu = [
+    <React.Fragment>
+      <NavDropDownButton
+        onToggle={() => {
+          setIsOpen2([false]);
+          setIsOpen1([!isOpen1[0]]);
+        }}
+        menuId="testDropDownOne"
+        isOpen={isOpen1[0]}
+        label={t('nav.dropdownOne.label')}
+        // isCurrent={true} // TODO: update later
+      />
+      <Menu
+        key="one"
+        items={testMenuItemsOne}
+        isOpen={isOpen1[0]}
+        id="testDropDownOne"
+      />
+    </React.Fragment>,
+    <React.Fragment>
+      <NavDropDownButton
+        onToggle={() => {
+          setIsOpen1([false]);
+          setIsOpen2([!isOpen2[0]]);
+        }}
+        menuId="testDropDownTwo"
+        isOpen={isOpen2[0]}
+        label={t('nav.dropdownTwo.label')}
+        // isCurrent={true} // TODO: update later
+      />
+      <Menu
+        key="two"
+        items={testMenuItemsTwo}
+        isOpen={isOpen2[0]}
+        id="testDropDownTwo"
+      />
+    </React.Fragment>,
+    <a href="#three" key="three" className="usa-nav__link"> {/* TODO: doesn't like when I use NavLink instead */}
```

Comment:
I think we should be able to avoid anchor tags. What is the error for NavLink?

Reply:
Fixed now. For some reason, it showed there was a `NavLink` component in `@trussworks/react-uswds` that I could use but it kept giving me errors. Ended up switching to the `NavLink` component in `react-router-dom`.